tests/test_libcomxml.py

```diff
@@ -7,7 +7,28 @@
 import locale
 import re
 from libcomxml.core import XmlField, XmlModel, clean_xml
+import os
+import sys
 
+_ROOT = os.path.abspath(os.path.dirname(__file__))
+
+
+if sys.version_info[0] < 3:
+    # python2
+    def assertXmlEqual(self, got, want):
+        from lxml.doctestcompare import LXMLOutputChecker
+        from doctest import Example
+
+        checker = LXMLOutputChecker()
+        if checker.check_output(want, got, 0):
+            return
+        message = checker.output_difference(Example("", want), got, 0)
+        raise AssertionError(message)
+
+    unittest.TestCase.assertXmlEqual = assertXmlEqual
+else:
```

Comment:
Why is only used in Python2? In switching branch port to python3 this works also in Python3